src/Administration/Resources/app/administration/package.json

```
@@ -48,6 +48,7 @@
     "safari >= 7"
   ],
   "dependencies": {
+    "@amplitude/analytics-browser": "^2.23.5",
```

I'd suggest pinning it to specific version and use the newest one
```suggestion
    "@amplitude/analytics-browser": "2.24.2",
```
https://www.npmjs.com/package/@amplitude/analytics-browser/v/2.22.0?activeTab=versions